README.md

```diff
@@ -277,7 +277,6 @@ To reduce the size of the installation from about 800 MByte down to about 200 MB
 
 -   Build error around `node-aead-crypto`
     -   node-gyp has been seen failing on MacOS
-    -   try node 10+, which does not require the crypto polyfill
```

Comment:
Shall we remove the entire entry?

Reply:
I'm not even sure if the workaround is still relevant... I am ok with removing the entire entry!